samples/file/wasm-app/main.c

```diff
@@ -12,7 +12,8 @@
 #include <sys/stat.h>
 #include <unistd.h>
 
-#define PATH_TEST_FILE "test.txt"
+#define PATH_TEST_FOLDER "/test"
+#define PATH_TEST_FILE (PATH_TEST_FOLDER "/test.txt")
```

Comment:
Operating on folder and file under `/` may require root privilege, using current directory or temporarily directory may be a better way